agent/util-scripts/unittests

```diff
@@ -615,6 +615,8 @@ declare -A post_hooks=(
     [test-07]='cat ${_testdir}/42-iter/sample42/tools-foobar/testhost.example.com/postprocess.log >> ${_testout} 2>&1'
     [test-19]='rm ${_testopt}/unittest-scripts/pbench-tool-meister-client'
     [test-53]='sort_testlog; sort_tmlogs; filter_tmerrs'
+    [test-54]='sed -i "s/optional arguments:/options:/" ${_testout}'
+    [test-55]='sed -i "s/optional arguments:/options:/" ${_testout}'
```

Comment:
It might be (a bit) safer to anchor the search string with "^", though an incidental match is unlikely.